src/main/java/com/google/sps/servlets/SendRatingEmailServlet.java

```diff
@@ -64,7 +68,7 @@ public boolean sendRatingEmailToStudent(String email) {
         Session session = Session.getDefaultInstance(props, null);
 
         String subject = "Rate Your Tutoring Session";
-=        String message = "Hi there,\n" +
+        String message = "Hi there,\n" +
```

Comment:
Is it possible to get the student's name and use it in this message? ('Hi [student name]' rather than 'Hi there')

Reply:
Yes, I can work on doing this.